experimental/Schemes/singular_locus.jl

```diff
@@ -121,6 +308,12 @@ end
 # Additional functionality for fractions                               #
 ########################################################################
 
+########################################################################
+# The following functions (derivative, jacobi_matrix) should go to 
+# mpoly-localizations.jl and mpolyquo-localizations.jl
+# QUESTION:
+# documentation for all MPolyAnyRingElem should be happen in to mpoly.jl 
```

Comment:
I think this is such a general command that it is not wanted to document it for all special concrete instances. It is documented for `MPolyElem` and can then be expected to work for all other ring elements where it makes sense in the same way. 

If we were documenting all of them, the user will only get a big salad when typing `?derivative`. Imagine documenting `Base.in` for every concrete instance...

Reply:
That was also my thought, when I wrote this question, but I wanted to be sure.